adal/devApps/XFormsApp/AppConstants.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace XFormsApp
+{
+    public static class AppConstants
+    {
+        //Applications
+        public static readonly string UiAutomationTestClientId = "3c1e0e0d-b742-45ba-a35e-01c664e14b16";
```

Comment:
Use 'const' instead, it's like 'static readonly' but cannot be initialized from the ctor.